packages/storage-driver-s3/src/index.ts

```diff
@@ -79,10 +84,10 @@ export class DriverS3 implements TusDriver {
 		 * often in rapid succession, hitting the maxSockets limit of 50.
 		 * The requestHandler is customized to get around this.
 		 */
-		const connectionTimeout = 5000;
-		const socketTimeout = 120000;
-		const maxSockets = 500;
-		const keepAlive = true;
+		const connectionTimeout = ms(String(this.config.connectionTimeout ?? 5000));
+		const socketTimeout = ms(String(this.config.socketTimeout ?? 120000));
+		const maxSockets = this.config.maxSockets ?? 500;
+		const keepAlive = this.config.keepAlive ?? true;
```

Comment:
I think typecasting might be needed here as the incoming env should of string type 🤔

Reply:
We can, but the env vars are already automatically converted to the most approximate type using the [guessType utility](https://github.com/joselcvarela/directus/blob/packages/storage/connection-configs/packages/env/src/utils/guess-type.ts#L3-L24)

Reply:
As this is a package, I believe if we want to cast, that should happen on API and not on the package itself, otherwise all configurations should also be strings in my opinion 🤔

Reply:
Ooo yes!  💯 on the guessType! Thanks~